app/assets/stylesheets/components/_form.sass

```diff
@@ -0,0 +1,41 @@
+$Form-border-input: Theme-color(input, border)
+$Form-color-filled: Theme-color(input, filled)
+$Form-color-input-filled: Theme-color(input, filled)
+$Form-color-input-placeholder: Theme-color(input, placeholder)
+$Form-color-label: Theme-color(text, label)
+$Form-fontSize-input: Theme-fontSize(text, base, font-size)
+$Form-height: 40px
+$Form-input-border: 2px
+$Form-input-width-hashtag: 212px
+$Form-spacing-base: $Theme-spacing-base
+$Form-spacing-small: $Theme-spacing-small
+$Form-spacing-tiny: $Theme-spacing-small / 2
+$Form-width: 425px
+
+.Form
```

Comment:
Basically each input should be its own component where you would set their style. This class has too many responsibilities and makes it hard to reuse